src/harness/sas_interface.py

```diff
@@ -178,6 +177,39 @@ def InjectFccId(self, request):
     pass
 
   @abc.abstractmethod
+  def InjectEscZone(self, request):
+    """SAS admin interface to inject EscZone information into SAS under test.
+    Args:
+      request: ey-value pair where the key is
+        "zone" and the value is  he information of the zone of type ZoneData as specified in specified in SAS-SAS TS:
+		id	string with format zone/$CREATOR/$ZONE_ID
```

Comment:
I think it's fine to omit this additional information. Anyone reading this code will have easy access to the SAS-SAS TS so including it won't save them much trouble. Having short comments will make it easier for someone to read through the file and find what they're looking for. It also means that we don't have to update the comment if the TS changes.